packages/@react-aria/grid/src/useGridCell.ts

```diff
@@ -96,6 +96,8 @@ export function useGridCell<T, C extends GridCollection<T>>(props: GridCellProps
       return;
     }
 
+    // TODO: Can't do {tabbable: true} because that would stop focus from reaching the roving tabindex -1 buttons in a action group
+    // within a grid cell
     let walker = getFocusableTreeWalker(ref.current);
```

Comment:
Still poking around to figure out a good way to remove the checkbox from the arrow key nav